src/core/ee/cop1.cpp

```diff
@@ -77,6 +77,11 @@ uint32_t Cop1::get_gpr(int index)
     return gpr[index].u;
 }
 
+float Cop1::get_gpr_f(int index)
```

Comment:
Recommend marking function as `const`.

```suggestion
float Cop1::get_gpr_f(int index) const
```